core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -56,7 +56,6 @@ export class OperatorPanelComponent implements OnInit {
     threshold: 0.3,
     location: 0,
     distance: 100,
-    maxPatternLength: 32,
```

Comment:
in the newest version, it does not provide option to set the maxPatternLength, it will be 32 as the default.